js/mainjs/config.js

```diff
@@ -6,7 +6,8 @@ const Path = require('path');
 const defaultConfig = {
 	homePlugin:  'Overview',
 	siad: {
-    	path: Path.join(__dirname, '../../', 'Sia'),
+		path: Path.join(__dirname, '../..', 'Sia', 'siad'),
+		datadir: Path.join(__dirname, '../..', 'Sia'),
```

Comment:
also, I think `Path.resolve` is preferred over `Path.join` here, though the difference is subtle